stream/.csslintrc

```diff
@@ -2,7 +2,7 @@
   "adjoining-classes": false,
   "box-model": 2,
   "box-sizing": 2,
-  "bulletproof-font-face": 2,
+  "bulletproof-font-face": false,
```

Comment:
This rule ensures that the font-face declaration will support IE<=8 -- I read in slack not to worry about IE8 support, and the approach proposed by this PR does not support it.